frontend/src/stores/useProjectStore.ts

```diff
@@ -24,7 +24,12 @@ import {
   DEFAULT_PROJECT_COLOR
 } from '../config/projects'
 
-export const createNewProject = (projectType: ProjectType = DEFAULT_PROJECT_TYPE): Project => ({
+/**
+ * A stored project has an extra `_id` field which is used to tell identify it
+ */
+type StoredProject = Project & {_id: string}
```

Comment:
Projects used by the store require an id but don't require one outside. This was the name I came up with but open for suggestions of a better name

Reply:
SGTM 👍🏻